uk.co.compucorp.civicrm.civihr/info.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<extension key="uk.co.compucorp.civicrm.civihr" type="module">
+  <file>civihr</file>
+  <name>CiviHR</name>
+  <description>Provides current CiviHR release details</description>
```

Comment:
I think this is a bit too specific. My idea was to have a core/base civihr extension where we could put all types of generic code that might be reused by other extensions, not only release details. What do you think?